x/application/keeper/application.go

```diff
@@ -83,12 +94,148 @@ func (k Keeper) GetAllApplications(ctx context.Context) (apps []types.Applicatio
 	return
 }
 
-// GetAllApplicationsIterator returns an iterator over all application records.
+// GetAllUnstakingApplicationsIterator returns an iterator for all applications
+// that are currently unstaking.
+// It is used to process applications that have completed their unbonding period.
```

Comment:
I'm confused whether it's applications that are **pending** or have **completed** unbonding. I'm assuming the former, but the second sentence seems contradictory. :thinking:

Reply:
It was to indicate that it's used by the endblocker to finalize their unbonding process.

But you're right, it's useless and misleading comment.